datarobot_batch_scoring/utils.py

```diff
@@ -372,10 +373,12 @@ def make_validation_call(user, api_token, n_retry, endpoint, base_headers,
             if user and api_token:
                 r = requests.post(endpoint, headers=base_headers,
                                   data=data,
-                                  auth=(user, api_token))
+                                  auth=(user, api_token),
+                                  verify=verify_ssl)
             elif not (user and api_token):
                 r = requests.post(endpoint, headers=base_headers,
-                                  data=data)
+                                  data=data,
+                                  verify=verify_ssl)
```

Comment:
I would rather that we expose the actual functionality of the `verify` parameter to be something where we can also allow users to point at a file that should be used to verify host identities, instead of just a boolean.

Reply:
agreed. I think it's already [takes path or bool](http://docs.python-requests.org/en/master/user/advanced/?highlight=verify#ssl-cert-verification) so I'll make changes